src/hypergraph.jl

```diff
@@ -225,22 +327,72 @@ function add_vertex!(h::Hypergraph{T, V, E, D};
     ix
 end
 
+"""
```

Comment:
Once we have the alias:
`BasicHypergraph{T, D} =  Hypergraph{T,Nothing,Nothing,D} where {T<:Real, D<:AbstractDict{Int,T}}`
this is also not neeeded

Reply:
I'm not even sure that alias is particularly helpful.

Reply:
Yes maybe let's skip it for now and decide later once the code is cleared :)